Library/Homebrew/rubocops/bottle.rb

```diff
@@ -0,0 +1,55 @@
+# typed: true
+# frozen_string_literal: true
+
+require "rubocops/extend/formula"
+
+module RuboCop
+  module Cop
+    module FormulaAudit
+      # This cop audits the `bottle` block in formulae.
+      class Bottle < FormulaCop
+        extend AutoCorrector
+
+        def audit_formula(_node, _class_node, _parent_class_node, body_node)
+          bottle_node = find_block(body_node, :bottle)
+          return if bottle_node.nil?
+
+          sha256_nodes = find_method_calls_by_name(bottle_node.body, :sha256)
+          cellar_node = find_node_method_by_name(bottle_node.body, :cellar)
+          cellar_source = cellar_node&.first_argument&.source
+
+          if sha256_nodes.present? && cellar_node.present?
+            offending_node(cellar_node)
+            problem "`cellar` should be a parameter to `sha256`" do |corrector|
+              corrector.remove(cellar_node.source_range)
+            end
+          end
+
+          sha256_nodes.each do |sha256_node|
+            sha256_hash = sha256_node.last_argument
+            sha256_pairs = sha256_hash.pairs
+            next if sha256_pairs.count != 1
+
+            sha256_pair = sha256_pairs.first
+            sha256_key = sha256_pair.key
+            sha256_value = sha256_pair.value
+            next unless sha256_value.is_a?(AST::SymbolNode)
```

Comment:
```suggestion
            next unless sha256_value.sym_type?
```

Are these equivalent? If so, I think I prefer `sym_type?` (although either is fine)

Reply:
Yes, I'm pretty sure they're equivalent. This specific file probably isn't affected, but generally Sorbet doesn't like the `*_type?` methods. Somewhat off-topic, but it'd be really cool if they ever eventually add support for this (`#sym_type` being equivalent to `#is_a?(SymbolNode)`)